tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+<?php
+/**
+* Plugin Name: Eventbrite Tickets Extension: Additional Options
+* Description: Adds a new Eventbrite options section to the bottom of wp-admin > Events > Settings > Imports tab. Options include text above or below iframe, iframe height, moving ticket location on Single Event view, displaying tickets for Private Eventbrite events, change API URL (e.g. from .com to .co.uk), and more.
+* Version: 1.0
+* Extension Class: Tribe__Extension__Eventbrite_Addl_Opts
+* Author: Modern Tribe, Inc.
+* Author URI: http://m.tri.be/1971
+* License: GPLv2 or later
+* License URI: https://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+* Extension main class, class begins loading on init() function.
+*/
+class Tribe__Extension__Eventbrite_Addl_Opts extends Tribe__Extension {
+
+	protected $opts_prefix = 'tribe_ext_eventbrite_opts_';
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 *
+	 * @access protected
+	 */
+	protected function construct() {
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Events__Main', '4.4' ); // due to using tribe() from Tribe-Common as of 4.4
+		$this->add_required_plugin( 'Tribe__Events__Tickets__Eventbrite__Main', '4.4' ); // just to match version of Tribe__Events__Main
+
+		// Set the extension's TEC URL
+		$this->set_url( 'https://theeventscalendar.com/extensions/eventbrite-additional-options/' );
+	}
+
+	/**
+	 * Adds settings options
+	 */
+	public function add_settings() {
+		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		$fields = array(
+			$this->opts_prefix . 'heading' => array(
+				'type' => 'html',
+				'html' => '<h3>' . esc_html__( 'Eventbrite Additional Options', 'tribe-extension' ) . '</h3>',
+			),
+			$this->opts_prefix . 'helper_text' => array(
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'The following fields are from an Extension. These additional Eventbrite options allow you to change how the Eventbrite Tickets add-on operates, such as moving the ticket location on Single Event view, displaying tickets for Private Eventbrite events, and more.', 'tribe-extension' ) . '</p>',
+			),
```

Comment:
Keeping the settings area as clean as possible has very appealing psychological effects. So we generally try to be very succinct, and only convey necessary information.

The above paragraph can be removed. If you review our settings pages you'll note we only use paragraphs like this in very specific circumstance, and this would not apply.